frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -87,65 +111,84 @@ export function SegmentStatisticsModal({
 }: Props) {
   const magInfo = getResolutionInfo(visibleSegmentationLayer.resolutions);
   const layersFinestResolution = magInfo.getFinestResolution();
-  const dataSetScale = Store.getState().dataset.dataSource.scale;
+  const dataSetScale = useSelector((state: OxalisState) => state.dataset.dataSource.scale);
+  const additionalCoordinates = useSelector(
+    (state: OxalisState) => state.flycam.additionalCoordinates,
+  );
+  const hasAdditionalCoords = hasAdditionalCoordinates(additionalCoordinates);
+  const additionalCoordinateStringForModal = getAdditionalCoordinatesAsString(
+    additionalCoordinates,
+    ", ",
+  );
   const dataSource = useFetch(
     async () => {
       await api.tracing.save();
-      const segmentStatisticsObjects = await Promise.all([
+      const segmentStatisticsObjects: Array<SegmentInfo> | string = await Promise.all([
```

Comment:
I would change this to
```suggestion
      const segmentStatisticsObjects: Array<SegmentInfo> | null = await Promise.all([
```

with `null` being the error case. the error message is clear in that case (it should be ERROR_CODE) and doesn't need to be contained here.

Reply:
ah, because the default value is `[]` and not `null`. thank you!